components/scrapd-grid.js

```diff
@@ -132,6 +133,30 @@ class ScrapdGrid extends React.Component {
                     .endOf('year')
                 ],
                 'This year': [moment().startOf('year'), moment().endOf('year')],
+                '2017': [
+                  moment()
+                    .startOf('year')
+                    .year(2017),
+                  moment()
+                    .endOf('year')
+                    .year(2017)
+                ],
+                '2018': [
+                  moment()
+                    .startOf('year')
+                    .year(2018),
+                  moment()
+                    .endOf('year')
+                    .year(2018)
+                ],
+                '2019': [
+                  moment()
+                    .startOf('year')
+                    .year(2019),
+                  moment()
+                    .endOf('year')
+                    .year(2019)
+                ],
```

Comment:
Can you remove `last year` and `this year` too please? They are not needed if we have the year numbers.

<img width="155" alt="image" src="https://user-images.githubusercontent.com/6969134/56700557-93be7f80-66c0-11e9-9676-a7d6961c7d8a.png">